_includes/custom-icon-links.html

```diff
@@ -1 +1,9 @@
-<!-- Optional additional links to insert for icons links -->
\ No newline at end of file
+<!-- Optional additional links to insert for icons links -->
+{% if site.hydeout.discord %}
+<a id="github-link"
+   class="icon" title="Discord Server" aria-label="Discord Server"
+   href="{{ site.hydeout.discord }}">
+  {% include svg/discord.svg %}
+  <span class="label">Discord Server</span>
+</a>
+{% endif %}
```

Comment:
Maybe move this with the other pink links, it makes a bit more sense since the rest of the blue ones are features of the site itself, if that makes sense. In the original mockup, both of the blue links were "external" things, but that isn't the case anymore since Jekyll has some features we need to expose. I don't know what the icon situation should be then, though.

Reply:
Yeah if we're gonna put it in navigation links I can't think of a good place to put Discord icon.

Reply:
Are we gonna keep this until we find a sensible solution?

Reply:
Yeah just leave it like this, it's good enough. I doubt anyone will actually care lol